client/fire-webclient/src/main/kotlin/uk/ac/ebi/fire/client/api/S3JClient.kt

```diff
@@ -3,21 +3,24 @@ package uk.ac.ebi.fire.client.api
 import com.amazonaws.services.s3.AmazonS3
 import com.amazonaws.services.s3.model.GetObjectRequest
 import com.amazonaws.services.s3.model.S3Object
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
 import uk.ac.ebi.fire.client.integration.web.FireS3Client
 import java.io.File
 import java.nio.file.Files
 import java.nio.file.StandardCopyOption
 
-class S3Client(
+class S3JClient(
```

Comment:
Are you considering adding a feature flag in another PR to choose between one client or the other? Otherwise, we could delete this class since, after this change, it will only be used by tests